src/main/java/com/google/sps/managers/DatastoreManager.java

```diff
@@ -376,5 +376,35 @@ public User retrieveUserByUsername(String username) {
     user.setGamesCompletedWithTime(gamesCompletedWithTime);
     return user.build();
   }
+  
+  /**
+  * Creates or Replaces the identification of a user in datastore as an entity
+  * @param email a String variable representing the user's email address
+  * @param id a String variable representing the user's id
+  */
+  public void createOrReplaceIdentification(String email, String id) {
+    Entity identificationEntity = new Entity ("Identification", email);
+    identificationEntity.setProperty("id", id);
+    identificationEntity.setProperty("Email", email);
```

Comment:
I think you can delete this line because you can already get the email due to line 386. I won't need to use this.

Reply:
Sure.